docs/GettingStartedDocs/IntelSDKPortingGuide.md

```diff
@@ -0,0 +1,214 @@
+  COMMAND openssl rsa -in private.pem -pubout -out public.pem)
+
+# Sign enclave
+add_custom_command(
+  OUTPUT MyEnclave.signed
+  DEPENDS MyEnclave MyEnclave.conf private.pem
+  COMMAND openenclave::oesign sign -e $<TARGET_FILE:MyEnclave>
+          -c ${CMAKE_SOURCE_DIR}/MyEnclave.conf -k private.pem)
+```
+
+Audiences are encouraged to look for complete examples under */path/to/OpenEnclave_InstallDir*/share/openenclave/samples directory.
+
+## Migrate Enclave Settings
+
+**Enclave Settings**, also known as **Enclave Metadata**, refer to information consumed by enclave loaders to instantiate enclaves, such as heap size, stack size, number of trusted hardware threads (i.e. number of TCS's), etc.
+
+Enclave settings are specified in human readable text formats by enclave developers, referred to as *configuration files*. Both Intel SGX SDK and OpenEnclave SDK provide tools to compile configuration files into their binary form and embed them into the final enclave image. However, they differ in both format and feature set.
+
+### Configuration File Formats
+
+Intel SGX SDK adopted an XML format for encoding enclave settings in text form, which is usually named as *Enclave*.config.xml. The signing tool (i.e. `sgx_sign`) converts it into binary form and stores it in a dedicated section (i.e. `.sgxmeta`) of the enclave's ELF image before it calculates the enclave's measurement (i.e. `SIGSTRUCT::MRENCLAVE`). At runtime, the ELF section `.sgxmeta` is consumed by the enclave loader to instatiate the exact enclave that matches the measurement caculated by the signing tool.
+
+Below comes from the sample code - SampleEnclave, of Intel SGX SDK.
+```xml
+<EnclaveConfiguration>
+  <ProdID>0</ProdID>
+  <ISVSVN>0</ISVSVN>
+  <StackMaxSize>0x40000</StackMaxSize>
+  <HeapMaxSize>0x100000</HeapMaxSize>
+  <TCSNum>10</TCSNum>
+  <TCSPolicy>1</TCSPolicy>
+  <!-- Recommend changing 'DisableDebug' to 1 to make the enclave undebuggable for enclave release -->
+  <DisableDebug>0</DisableDebug>
+  <MiscSelect>0</MiscSelect>
+  <MiscMask>0xFFFFFFFF</MiscMask>
+</EnclaveConfiguration>
+```
+
+Rather than XML, OpenEnclave SDK uses plaintext files instead. A configuration file, usually named as *enclave*.conf, is supplied to the signing tool (i.e. `oesign`) command line to govern the instantiation of the enclave. The compiled metadata is then stored in a dedicated ELF section named `.oeinfo`. Additionally, OpenEnclave SDK provides `OE_SET_ENCLAVE_SGX`, a C macro for embedding default enclave settings in C source files. *enclave*.conf is in fact optional and is necessary only if some of those defaults provided to `OE_SET_ENCLAVE_SGX` macro need overridden. Under the hood, `OE_SET_ENCLAVE_SGX` is expanded to instantiation of an `oe_sgx_enclave_properties_t` structure in `.oeinfo` section. Detailed information can be found in [buildandsign.md](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/buildandsign.md) in OpenEnclave SDK documentation.
```

Comment:
```suggestion
Rather than XML, OpenEnclave SDK uses plaintext files instead. A configuration file, usually named as *enclave*.conf, is supplied to the signing tool (i.e., `oesign`) command line to govern the instantiation of the enclave. The compiled metadata is then stored in a dedicated ELF section named `.oeinfo`. Additionally, OpenEnclave SDK provides `OE_SET_ENCLAVE_SGX`, a C macro for embedding default enclave settings in C source files. *enclave*.conf is in fact optional and is necessary only if some of those defaults provided to `OE_SET_ENCLAVE_SGX` macro need overridden. Under the hood, `OE_SET_ENCLAVE_SGX` is expanded to instantiation of an `oe_sgx_enclave_properties_t` structure in `.oeinfo` section. Detailed information can be found in [buildandsign.md](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/buildandsign.md) in OpenEnclave SDK documentation.
```
Also please split into multiple source lines so you don't have to visually play "spot the differences" in suggestions like above.